src/sql/workbench/services/dialog/browser/dialogModal.ts

```diff
@@ -56,7 +56,11 @@ export class DialogModal extends Modal {
 
 	public render() {
 		super.render();
-		attachModalDialogStyler(this, this._themeService);
+		if (this._modalOptions.dialogStyle === 'callout') {
```

Comment:
I feel like a helper function `attachModalDialogTypeStyler` or something would be good to have that would attach the correct styler based on the type of modal. If we add more types in the future it'd be easy to miss updating some of these

Reply:
This feature has been in the works for months now -- do we want to implement this now or create a followup task?

Reply:
The ask here isn't very big - and considering that you're just now adding this special type of styler based on the type of modal dialog I don't think it's unreasonable to do now.

Reply:
Agreed. How/where should I begin?

Reply:
Added conditional within styler to provide colors for dialogModal depending on dialogStyle